Assets/Plugins/Editor/Uplift/LogAggregator.cs

```diff
@@ -53,6 +53,7 @@ class LogHandlerUtils {
 
 	class LogAggregator : ILogHandler, IDisposable
 	{
+		//TODO reactivate Log aggregator
```

Comment:
where was it deactivated?

Reply:
I deactivated it the the first commit ["Lots of debug log"](https://github.com/DragonBox/uplift/pull/84/commits/48dee360a847cafd9ad7f5d143051a0cb7195d61), and was reactivated in ["Reactivate log aggregator which was disabled for debugging purposes"](https://github.com/DragonBox/uplift/pull/84/commits/c55c9f677778098ae792c5e777262c3c8a204eb4l).

I forgot to mention I deactivated it in the first commit, my bad.